src/routes/github/subscription-deferred-install/github-subscription-deferred-install-post.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response } from "express";
+import {
+	extractSubscriptionDeferredInstallPayload,
+	forgetSubscriptionDeferredInstallRequest
+} from "services/subscription-deferred-install-service";
+import { verifyAdminPermsAndFinishInstallation } from "services/subscription-installation-service";
+
+export const GithubSubscriptionDeferredInstallPost = async (req: Request, res: Response) => {
+	const payload = await extractSubscriptionDeferredInstallPayload(req);
+	const { githubToken, installation } = res.locals;
+
+	const result = await verifyAdminPermsAndFinishInstallation(
+		githubToken, installation, payload.gitHubServerAppIdPk, payload.gitHubInstallationId, req.log
+	);
+	if (result.error) {
+		res.status(401).json(result);
+		return;
+	}
+
+	await forgetSubscriptionDeferredInstallRequest(req);
+
+	res.status(200).json({ ok: true });
```

Comment:
What user see? I think a more detail message, even if just json, will be better for `GitHub Admin`, like `Thank you have just connect xxx to xxx successfully blah blah`

Reply:
Nit: success: true (to align with responses in the rest of our middleware and node/express common conventions) + add a message `{ success: true, message: "Something like what Gary said or whatever tickles your fancy" }